README.md

```diff
@@ -74,6 +74,8 @@ manually if your distribution does not provide a package.
 
 First, make sure you have the required __dependencies__:
 - build:
+	- autopoint (for bootstrapping)
```

Comment:
IMO, I think we should stick to upstream. Autopoint is part of gettext: http://git.savannah.gnu.org/cgit/gettext.git/tree/gettext-tools/misc/autopoint.in
and autoreconf is part of autoconf: http://git.savannah.gnu.org/cgit/autoconf.git/tree/bin/autoreconf.in

Both are already listed. Distro-specific notes should be added separately.